src/datasource/typeorm.module.ts

```diff
@@ -1,6 +1,6 @@
 import { Global, Module } from '@nestjs/common';
 import { DataSource } from 'typeorm';
-
+// Add to review
```

Comment:
Crea separatamente la factory in un file ad hoc e la importi, così come il provider.